middleware/handleGet.js

```diff
@@ -6,17 +6,19 @@ const {
   getTopWinStreaks,
 } = require('./crud-functions/index.js');
 const {orderObjects} = require('../utilities/object-formatting.js');
-const orderedColumns = require('../constants');
+const {orderedColumns, commands} = require('../constants');
 
 async function handleGet(ctx) {
   const {text} = ctx.request.body;
 
-  let [category, numUsers] = text.split(' ');
-  numUsers = parseInt(numUsers);
+  const command = commands.find((command) => text.includes(command)) || null;
+  const numUsers = Number(text.match(/\d+/));
```

Comment:
This finds the first sequence of numbers in the text. Do we want that? Something like `"3 wins 4 losses 5"` would return `wins` as command and `3` as number of users. I think we can make the regex stricter by only accepting the `${command} ${numUsers}` format and throw an error if it's not in the format we want, but that depends on how much time we want to spend on this hackdays project.

Reply:
good point, I'll change it back.